.setup/install_system.sh

```diff
@@ -435,6 +435,13 @@ if [ ${WORKER} == 0 ]; then
 
     a2enmod include actions cgi suexec authnz_external headers ssl proxy_fcgi rewrite proxy_http proxy_wstunnel
 
+    # Disable mod_php and prefork based multiprocessing module
+    # as Submitty uses php-fpm to handle the PHP requests
+    a2dismod php${PHP_VERSION} mpm_prefork
+    # Enable event based multiprocessing module and http/2 module
+    # this will enable http/2 on any directory, if possible
+    a2enmod mpm_event http2
```

Comment:
Won't this then require that servers have SSL/TLS enabled already? I'm only pointing this out cause it would effect new vagrant machines. Maybe this should be wrapped in a vagrant == 0?

Reply:
I'll remove these lines, as the upgrade now is a part of sysadmin actions.  Thank you for reviewing.